src/routing/core.rs

```diff
@@ -1260,6 +1260,29 @@ impl Core {
                 (MIN_AGE + 1, None, None)
             };
 
+        // During the first section, node shall use ranged age to avoid too many nodes got
+        // relocated at the same time. After the first section got split, later on nodes shall
+        // only start with age of MIN_AGE + 1
+        if self.section.prefix().is_empty() {
+            if peer.age() < FIRST_SECTION_MIN_AGE || peer.age() > FIRST_SECTION_MAX_AGE {
+                debug!(
+                    "Ignoring JoinRequest from {} - first-section node having wrong age {:?}",
+                    peer,
+                    peer.age(),
+                );
+                return Ok(vec![]);
+            } else {
+                age = peer.age();
+            }
+        } else if peer.age() != MIN_AGE + 1 {
```

Comment:
Can this be a case? New nodes with `age=MIN_AGE+1` can join after startup no?

Reply:
After section split, new node `can only` join with age of MIN_AGE+1.
But this reminds me the relocated node shall be the exception.
I will push a fix commit to fix this.

Reply:
Best to identify Infant here as age=4 or joining infant as age = 5 and use that const. That way we keep code simple to read.

Reply:
changed to use the const of MIN_ADULT_AGE